src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -1,49 +1,67 @@
 import javax.swing.*
 
 /**
  * This class stores the UI of the TestGenie tool window.
  */
 class TestGenieToolWindow {
 
-    private var searchBudget: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
-    private var localSearchBudgetType: ComboBox<LocalSearchBudgetType> = ComboBox(LocalSearchBudgetType.values())
-    private var localSearchBudgetValue: JSpinner = JSpinner(SpinnerNumberModel(5, 0, 10000, 1))
+    // UI elements for EvoSuite parameters
     private var stoppingCondition: ComboBox<StoppingCondition> = ComboBox<StoppingCondition>(StoppingCondition.values())
+    private var searchBudget: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
     private var initializationTimeout: JSpinner = JSpinner(SpinnerNumberModel(120, 0, 10000, 1))
     private var minimisationTimeout: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
     private var assertionTimeout: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
     private var junitCheckTimeout: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
-    private var population: JSpinner = JSpinner(SpinnerNumberModel(50, 0, 10000, 1))
     private var populationLimit: ComboBox<PopulationLimit> = ComboBox(PopulationLimit.values())
+    private var population: JSpinner = JSpinner(SpinnerNumberModel(50, 0, 10000, 1))
 
-    private val panelTitle = JLabel("Frequently Used Parameters")
+    // Save and Reset buttons
     private var saveButton: JButton = JButton("Save")
     private var resetButton: JButton = JButton("Reset")
 
+    // Tool Window panel
+    private val panelTitle = JLabel("Quick Access Parameters")
     private var toolWindowPanel: JPanel = JPanel()
 
+    // The tooltip labels
+    private var stoppingConditionToolTip = JBLabel("Default: 60 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
+    private var populationLimitToolTip = JBLabel("Default: 60 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
+
+    // Template strings for "default" tooltips
     private val defaultStr: String = "Default: %s"
 
+
     init {
         loadState()
 
         toolWindowPanel = createToolWindowPanel()
-
         panelTitle.font = Font("Monochrome", Font.BOLD, 20)
-        resetButton.toolTipText = "Reset all parameters to their default values"
+
+        addTooltipsToUiElements()
+
+        stoppingConditionToolTip.border = JBUI.Borders.emptyLeft(10)
+        stoppingCondition.addActionListener {
+            stoppingConditionToolTip.text = default("60 ${stoppingCondition.item.units()}")
+        }
+        stoppingConditionToolTip.text = default("60 ${stoppingCondition.item.units()}")
+
+        populationLimitToolTip.border = JBUI.Borders.emptyLeft(10)
+        populationLimit.addActionListener {
+            populationLimitToolTip.text = default("60 ${populationLimit.item.toString().toLowerCase()}")
+        }
+        populationLimitToolTip.text = default("60 ${populationLimit.item.toString().toLowerCase()}")
```

Comment:
There seems to be a little bit of code duplication here. Could that be fixed somehow?

Reply:
This comment was addressed by Jegor and you in the code. I think that everything should be fine now.